tracing/src/main/java/io/strimzi/test/tracing/TracingUtil.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.tracing;
+
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.kafka.streams.KafkaClientSupplier;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Tracing initialization
+ */
+public class TracingUtil {
+    private static final Logger log = LogManager.getLogger(TracingUtil.class);
+    private static TracingHandle tracing = new NoopTracing();
+
+    private static final List<TracingHandle> HANDLES = List.of(
+        new OpenTracingHandle(),
+        new OpenTelemetryHandle()
+    );
+
+    public static TracingHandle getTracing() {
+        return tracing;
+    }
+
+    public static TracingHandle initialize() {
+        for (TracingHandle instance : HANDLES) {
+            String serviceName = instance.serviceName();
+            if (serviceName != null) {
+                log.info(
+                    String.format(
+                        "Initializing Jaeger (%s) tracingConfig with service name %s",
+                        instance.type(),
+                        serviceName
+                    )
+                );
+                instance.initialize();
+                tracing = instance;
+                break;
+            }
+        }
+        return tracing;
+    }
+
+    private static final class NoopTracing implements TracingHandle {
+        @Override
+        public String type() {
+            return null;
+        }
+
+        @Override
+        public String envName() {
+            return null;
+        }
+
+        @Override
+        public String serviceName() {
+            return null;
+        }
+
+        @Override
+        public void initialize() {
+        }
+
+        @Override
+        public void kafkaConsumerConfig(Properties props) {
+        }
+
+        @Override
+        public void kafkaProducerConfig(Properties props) {
+        }
+
+        @Override
+        public KafkaClientSupplier clientSupplier() {
+            return null;
+        }
+    }
```

Comment:
Would make sense to do this:
- change the `TracingHandle` to be `abstract class`
- have there these `null`s
- `kafkaConsumerConfig` & `kafkaProducerConfig` are same for both tracing options, so having them in abstract would save the overrides
- then instead of `private static TracingHandle tracing = new NoopTracing();` you would do just `    private static TracingHandle tracing = new TracingHandle()` and you don't have to do another class in class?

Reply:
It's just another no-impl class, no real overhead ... and it clearly shows a no-op.
Actual impl have all methods different -- `kafkaConsumerConfig` & `kafkaProducerConfig` are different, just same class name, diff package.
I prefer interfaces over actual classes -- to point out the needed signature.
But if you insist on this, np ...

Reply:
It was just my point, if it makes more sense to have it like you have, then keep it as it is.

Reply:
Yeah, not a bad point -- when things are duplicated, which is not the case here -- things looks similar (since they are all tracing, no magic there ...), but differ in impl details (or packages).
We could have a default impl in the interface (nulls, no-op, etc) or an abstract class with these defaults.
But it doesn't hurt to be more explicit with Noop impl.

Reply:
I left the Noop ...